ChangeLog

```diff
@@ -3,6 +3,19 @@ Pylint's ChangeLog
 ------------------
 
 
+What's New in Pylint 2.15.0?
```

Comment:
There's going to be some change to the changelog/whatsnew in 2.15, I'm trying to have only one changelog entry to add in #5728.